command-expansion/src/app.js

```diff
@@ -0,0 +1,11 @@
+"use strict";
```

Comment:
Since this `.js` file is built (generated by `tsc`) we can/should remove it

Reply:
Actually, this file shouldn't be here so I removed it.